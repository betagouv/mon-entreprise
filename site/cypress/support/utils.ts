export const fr = Cypress.env('language') === 'fr'
export const baseUrl = Cypress.config('baseUrl')
