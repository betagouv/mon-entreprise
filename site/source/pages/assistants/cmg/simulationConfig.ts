import { SimulationConfig } from '@/domaine/SimulationConfig'

export const configCMG: SimulationConfig = {}
