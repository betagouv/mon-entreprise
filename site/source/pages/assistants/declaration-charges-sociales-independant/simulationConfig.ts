import { SimulationConfig } from '@/domaine/SimulationConfig'

export const config: SimulationConfig = {
	'objectifs exclusifs': [
		'déclaration charge sociales . résultat . cotisations obligatoires',
		'déclaration charge sociales . résultat . total charges sociales déductible',
		'déclaration charge sociales . résultat . revenu net fiscal',
		'déclaration charge sociales . résultat . assiette sociale',
	],
	situation: {
		'déclaration charge sociales': 'oui',
		'dirigeant . régime social': "'indépendant'",
		date: '01/01/2024',
		'dirigeant . indépendant . PL . CIPAV': 'non',
		'entreprise . catégorie juridique': "''",
		impôt: 'non',
	},
	'unité par défaut': '€',
}
