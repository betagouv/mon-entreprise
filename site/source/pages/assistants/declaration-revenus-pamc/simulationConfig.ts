import { SimulationConfig } from '@/domaine/SimulationConfig'

export const configDéclarationRevenusPAMC: SimulationConfig = {
	situation: {
		'déclaration revenus PAMC': 'oui',
	},
	'unité par défaut': '€',
}
