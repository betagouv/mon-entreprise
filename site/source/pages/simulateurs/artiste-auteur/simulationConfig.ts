import { SimulationConfig } from '../_configs/types'

export const configArtisteAuteur: SimulationConfig = {
	objectifs: [
		'artiste-auteur . cotisations',
		'artiste-auteur . revenus',
		'artiste-auteur . cotisations . IRCEC',
	],
	situation: {
		'artiste-auteur': 'oui',
		dirigeant: {
			'applicable si': 'non',
		},
	},
	'unité par défaut': '€/an',
	questions: {
		liste: [
			'artiste-auteur . revenus . BNC . micro-bnc',
			'artiste-auteur . cotisations . option surcotisation',
			'artiste-auteur . cotisations . IRCEC . cotisation RAAP . taux réduit',
			'artiste-auteur . cotisations . IRCEC . profession',
		],
	},
}
