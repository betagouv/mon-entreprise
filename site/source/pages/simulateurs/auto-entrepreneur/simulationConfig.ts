import { SimulationConfig } from '../_configs/types'

export const configAutoEntrepreneur: SimulationConfig = {
	'objectifs exclusifs': [
		"dirigeant . auto-entrepreneur . chiffre d'affaires",
		'dirigeant . auto-entrepreneur . revenu net',
		'dirigeant . auto-entrepreneur . revenu net . après impôt',
	],
	objectifs: [
		'dirigeant . auto-entrepreneur . cotisations et contributions',
		'dirigeant . rémunération . impôt',
	],
	questions: {
		"à l'affiche": [
			{
				label: "Type d'activité",
				dottedName: 'entreprise . activité . nature',
			},
			{
				label: 'Versement libératoire',
				dottedName:
					'dirigeant . auto-entrepreneur . impôt . versement libératoire',
			},
			{
				label: 'Impôt sur le revenu',
				dottedName: 'impôt . méthode de calcul',
			},
			{
				label: 'ACRE',
				dottedName: 'dirigeant . exonérations . ACRE',
			},
		],
		liste: [
			'entreprise',
			'dirigeant',
			'impôt',
			'établissement',
			'situation personnelle',
		],
		'non prioritaires': ['établissement . commune'],
		'liste noire': [
			'entreprise . activités',
			'entreprise . activités . revenus mixtes',
			'entreprise . charges',
			"entreprise . chiffre d'affaires",
		],
	},
	'unité par défaut': '€/an',
	situation: {
		salarié: 'non',
		'entreprise . catégorie juridique': "'EI'",
		'entreprise . catégorie juridique . EI . auto-entrepreneur': 'oui',
	},
}
