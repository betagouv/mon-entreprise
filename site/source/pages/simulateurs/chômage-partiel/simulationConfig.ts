import { SimulationConfig } from '../_configs/types'

export const configChômagePartiel: SimulationConfig = {
	objectifs: [
		'salarié . contrat . salaire brut',
		'salarié . rémunération . net . à payer avant impôt',
		'salarié . activité partielle . net habituel',
		'salarié . activité partielle . total employeur habituel',
		'salarié . coût total employeur',
	],
	questions: {
		liste: [
			'salarié . activité partielle',
			'salarié . temps de travail',
			'établissement . commune',
		],
	},
	'unité par défaut': '€/mois',
	situation: {
		dirigeant: 'non',
		'salarié . activité partielle': 'oui',
		'salarié . cotisations . prévoyances': {
			'applicable si': 'non',
		},
	},
}
