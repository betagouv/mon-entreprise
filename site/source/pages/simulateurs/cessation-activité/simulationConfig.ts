import { SimulationConfig } from '@/pages/simulateurs/_configs/types'
import { configIndépendant } from '@/pages/simulateurs/indépendant/simulationConfig'

export const configCessationActivité: SimulationConfig = {
	...configIndépendant,
	'unité par défaut': '€/an',
}
