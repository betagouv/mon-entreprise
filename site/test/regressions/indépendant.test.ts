import { DottedName } from 'modele-social'
import { expect, it } from 'vitest'

import { configIndépendant } from '@/pages/simulateurs/indépendant/simulationConfig'

import independentSituations from './indépendant.yaml'
import { engine, getMissingVariables, runSimulations } from './utils'

it('calculate simulations-indépendant', () => {
	const objectifs = [
		'dirigeant . rémunération . totale',
		'dirigeant . rémunération . cotisations',
		'dirigeant . rémunération . net',
		'dirigeant . indépendant . revenu professionnel',
		'impôt . montant',
		'dirigeant . rémunération . net . après impôt',
		'entreprise . charges',
		"entreprise . chiffre d'affaires",
		'dirigeant . indépendant . cotisations et contributions . début activité',
	] as DottedName[]
	runSimulations(independentSituations, objectifs, configIndépendant.situation)

	expect(
		getMissingVariables(
			engine
				.setSituation(configIndépendant.situation)
				.evaluate('dirigeant . rémunération . net')
		)
	).toMatchInlineSnapshot(`
		[
		  "dirigeant . indépendant . IJSS",
		  "dirigeant . indépendant . conjoint collaborateur",
		  "dirigeant . indépendant . cotisations et contributions . exonérations . pension invalidité",
		  "dirigeant . indépendant . revenus étrangers",
		  "dirigeant . rémunération . net",
		  "entreprise . activité . nature",
		  "entreprise . activités",
		  "entreprise . activités . agricole",
		  "entreprise . activités . artisanale",
		  "entreprise . activités . commerciale",
		  "entreprise . activités . libérale",
		  "entreprise . activités . revenus mixtes",
		  "entreprise . activités . saisonnière",
		  "entreprise . activités . service ou vente",
		  "entreprise . associés",
		  "entreprise . catégorie juridique",
		  "entreprise . catégorie juridique . EI",
		  "entreprise . catégorie juridique . EI . auto-entrepreneur",
		  "entreprise . charges",
		  "entreprise . chiffre d'affaires",
		  "entreprise . chiffre d'affaires . vente restauration hébergement",
		  "entreprise . date de cessation",
		  "entreprise . date de création",
		  "entreprise . imposition . IR . type de bénéfices",
		  "impôt . foyer fiscal . revenu imposable . autres revenus imposables",
		  "impôt . méthode de calcul",
		  "situation personnelle . RSA",
		  "situation personnelle . domiciliation fiscale à l'étranger",
		  "établissement . commune . département",
		  "établissement . commune . département . outre-mer",
		]
	`)
})
