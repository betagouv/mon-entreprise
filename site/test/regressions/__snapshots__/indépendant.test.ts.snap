// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`calculate simulations-indépendant > DROM 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: undefined
dirigeant . rémunération . cotisations: undefined
dirigeant . rémunération . net: undefined
dirigeant . rémunération . net . après impôt: undefined
dirigeant . rémunération . totale: 20000
entreprise . charges: 0
entreprise . chiffre d'affaires: undefined
impôt . montant: undefined

Notifications affichées : dirigeant . indépendant . cotisations et contributions . exonérations . DROM"
`;

exports[`calculate simulations-indépendant > acre 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: 1587
dirigeant . indépendant . revenu professionnel: 51985
dirigeant . rémunération . cotisations: 23208
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 41118
dirigeant . rémunération . totale: 73208
entreprise . charges: 0
entreprise . chiffre d'affaires: 73208
impôt . montant: 8882

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > activité 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 20768
dirigeant . rémunération . cotisations: 8398
dirigeant . rémunération . net: 20000
dirigeant . rémunération . net . après impôt: 19359
dirigeant . rémunération . totale: 28398
entreprise . charges: 0
entreprise . chiffre d'affaires: 28398
impôt . montant: 641

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > activité 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 20768
dirigeant . rémunération . cotisations: 8416
dirigeant . rémunération . net: 20000
dirigeant . rémunération . net . après impôt: 19359
dirigeant . rémunération . totale: 28416
entreprise . charges: 0
entreprise . chiffre d'affaires: 28416
impôt . montant: 641

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > années précédentes 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 51982
dirigeant . rémunération . cotisations: 23100
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 40812
dirigeant . rémunération . totale: 73100
entreprise . charges: 0
entreprise . chiffre d'affaires: 73100
impôt . montant: 9188

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > années précédentes 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 41600
dirigeant . rémunération . cotisations: 19029
dirigeant . rémunération . net: 40000
dirigeant . rémunération . net . après impôt: 33598
dirigeant . rémunération . totale: 59029
entreprise . charges: 0
entreprise . chiffre d'affaires: 59029
impôt . montant: 6402

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > années précédentes 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: undefined
dirigeant . indépendant . revenu professionnel: undefined
dirigeant . rémunération . cotisations: undefined
dirigeant . rémunération . net: 30000
dirigeant . rémunération . net . après impôt: undefined
dirigeant . rémunération . totale: undefined
entreprise . charges: 0
entreprise . chiffre d'affaires: undefined
impôt . montant: undefined"
`;

exports[`calculate simulations-indépendant > conjoint collaborateur 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 52109
dirigeant . rémunération . cotisations: 27818
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 41081
dirigeant . rémunération . totale: 77818
entreprise . charges: 0
entreprise . chiffre d'affaires: 77818
impôt . montant: 8919

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > conjoint collaborateur 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 52124
dirigeant . rémunération . cotisations: 28372
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 41077
dirigeant . rémunération . totale: 78372
entreprise . charges: 0
entreprise . chiffre d'affaires: 78372
impôt . montant: 8923

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > conjoint collaborateur 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 52192
dirigeant . rémunération . cotisations: 30891
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 41056
dirigeant . rémunération . totale: 80891
entreprise . charges: 0
entreprise . chiffre d'affaires: 80891
impôt . montant: 8944

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > conjoint collaborateur 4`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 52016
dirigeant . rémunération . cotisations: 24406
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 41109
dirigeant . rémunération . totale: 74406
entreprise . charges: 0
entreprise . chiffre d'affaires: 74406
impôt . montant: 8891

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > conjoint collaborateur 5`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 52016
dirigeant . rémunération . cotisations: 24406
dirigeant . rémunération . net: 50000
dirigeant . rémunération . net . après impôt: 41109
dirigeant . rémunération . totale: 74406
entreprise . charges: 0
entreprise . chiffre d'affaires: 74406
impôt . montant: 8891

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > conjoint collaborateur 6`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 517934
dirigeant . rémunération . cotisations: 160626
dirigeant . rémunération . net: 500000
dirigeant . rémunération . net . après impôt: 281568
dirigeant . rémunération . totale: 660626
entreprise . charges: 0
entreprise . chiffre d'affaires: 660626
impôt . montant: 218432"
`;

exports[`calculate simulations-indépendant > cotisations facultatives 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 18064
dirigeant . rémunération . cotisations: 12747
dirigeant . rémunération . net: 17253
dirigeant . rémunération . net . après impôt: 17044
dirigeant . rémunération . totale: 30000
entreprise . charges: 0
entreprise . chiffre d'affaires: 30000
impôt . montant: 209

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations facultatives 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 17945
dirigeant . rémunération . cotisations: 13947
dirigeant . rémunération . net: 16053
dirigeant . rémunération . net . après impôt: 15863
dirigeant . rémunération . totale: 30000
entreprise . charges: 0
entreprise . chiffre d'affaires: 30000
impôt . montant: 190

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations facultatives 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 20864
dirigeant . rémunération . cotisations: 9947
dirigeant . rémunération . net: 20053
dirigeant . rémunération . net . après impôt: 19397
dirigeant . rémunération . totale: 30000
entreprise . charges: 0
entreprise . chiffre d'affaires: 30000
impôt . montant: 656

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations facultatives 4`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 14196
dirigeant . rémunération . cotisations: 6344
dirigeant . rémunération . net: 13656
dirigeant . rémunération . net . après impôt: 13656
dirigeant . rémunération . totale: 20000
entreprise . charges: 0
entreprise . chiffre d'affaires: 20000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations facultatives 5`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 225990
dirigeant . rémunération . cotisations: 82153
dirigeant . rémunération . net: 217847
dirigeant . rémunération . net . après impôt: 139007
dirigeant . rémunération . totale: 300000
entreprise . charges: 0
entreprise . chiffre d'affaires: 300000
impôt . montant: 78840"
`;

exports[`calculate simulations-indépendant > cotisations facultatives 6`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 222490
dirigeant . rémunération . cotisations: 85653
dirigeant . rémunération . net: 214347
dirigeant . rémunération . net . après impôt: 137082
dirigeant . rémunération . totale: 300000
entreprise . charges: 0
entreprise . chiffre d'affaires: 300000
impôt . montant: 77265"
`;

exports[`calculate simulations-indépendant > cotisations forfaitaires début d'activité 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: 3359
dirigeant . indépendant . revenu professionnel: 35533
dirigeant . rémunération . cotisations: 15822
dirigeant . rémunération . net: 34178
dirigeant . rémunération . net . après impôt: 30232
dirigeant . rémunération . totale: 50000
entreprise . charges: 0
entreprise . chiffre d'affaires: 50000
impôt . montant: 3946

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations minimales 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 136
dirigeant . rémunération . cotisations: 1338
dirigeant . rémunération . net: 100
dirigeant . rémunération . net . après impôt: 100
dirigeant . rémunération . totale: 1438
entreprise . charges: 0
entreprise . chiffre d'affaires: 1438
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations minimales 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 104
dirigeant . rémunération . cotisations: 156
dirigeant . rémunération . net: 100
dirigeant . rémunération . net . après impôt: 100
dirigeant . rémunération . totale: 256
entreprise . charges: 0
entreprise . chiffre d'affaires: 256
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > cotisations minimales 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 830
dirigeant . rémunération . cotisations: 437
dirigeant . rémunération . net: 800
dirigeant . rémunération . net . après impôt: 800
dirigeant . rémunération . totale: 1237
entreprise . charges: 0
entreprise . chiffre d'affaires: 1237
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > exonération pension invalidité 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 41487
dirigeant . rémunération . cotisations: 4732
dirigeant . rémunération . net: 40268
dirigeant . rémunération . net . après impôt: 34536
dirigeant . rémunération . totale: 45000
entreprise . charges: 0
entreprise . chiffre d'affaires: 45000
impôt . montant: 5732

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > exonération pension invalidité 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 7004
dirigeant . rémunération . cotisations: 2237
dirigeant . rémunération . net: 6763
dirigeant . rémunération . net . après impôt: 6763
dirigeant . rémunération . totale: 9000
entreprise . charges: 0
entreprise . chiffre d'affaires: 9000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > exonération pension invalidité 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: 970
dirigeant . indépendant . revenu professionnel: 9255
dirigeant . rémunération . cotisations: 1013
dirigeant . rémunération . net: 8987
dirigeant . rémunération . net . après impôt: 8987
dirigeant . rémunération . totale: 10000
entreprise . charges: 0
entreprise . chiffre d'affaires: 10000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > exonération pension invalidité 4`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 13358
dirigeant . rémunération . cotisations: 2046
dirigeant . rémunération . net: 12954
dirigeant . rémunération . net . après impôt: 12954
dirigeant . rémunération . totale: 15000
entreprise . charges: 0
entreprise . chiffre d'affaires: 15000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > exonération âge 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 39351
dirigeant . rémunération . cotisations: 17138
dirigeant . rémunération . net: 37862
dirigeant . rémunération . net . après impôt: 32770
dirigeant . rémunération . totale: 55000
entreprise . charges: 0
entreprise . chiffre d'affaires: 55000
impôt . montant: 5092

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > imposition à l'IS 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 71824
dirigeant . rémunération . cotisations: 30888
dirigeant . rémunération . net: 69112
dirigeant . rémunération . net . après impôt: 56433
dirigeant . rémunération . totale: 100000
entreprise . charges: 0
entreprise . chiffre d'affaires: 100000
impôt . montant: 12679

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > imposition à l'IS 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 71824
dirigeant . rémunération . cotisations: 30888
dirigeant . rémunération . net: 69112
dirigeant . rémunération . net . après impôt: 56433
dirigeant . rémunération . totale: 100000
entreprise . charges: 0
entreprise . chiffre d'affaires: 100000
impôt . montant: 12679

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > impôt sur le revenu 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 20768
dirigeant . rémunération . cotisations: 8398
dirigeant . rémunération . net: 20000
dirigeant . rémunération . net . après impôt: 17923
dirigeant . rémunération . totale: 28398
entreprise . charges: 0
entreprise . chiffre d'affaires: 28398
impôt . montant: 2077

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 628
dirigeant . rémunération . cotisations: 1423
dirigeant . rémunération . net: 577
dirigeant . rémunération . net . après impôt: 577
dirigeant . rémunération . totale: 2000
entreprise . charges: 0
entreprise . chiffre d'affaires: 2000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 42397
dirigeant . rémunération . cotisations: 19229
dirigeant . rémunération . net: 40771
dirigeant . rémunération . net . après impôt: 36038
dirigeant . rémunération . totale: 60000
entreprise . charges: 0
entreprise . chiffre d'affaires: 60001
impôt . montant: 4733

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 10382
dirigeant . rémunération . cotisations: 4192
dirigeant . rémunération . net: 10000
dirigeant . rémunération . net . après impôt: 10000
dirigeant . rémunération . totale: 14192
entreprise . charges: 0
entreprise . chiffre d'affaires: 14192
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 4`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 10382
dirigeant . rémunération . cotisations: 4192
dirigeant . rémunération . net: 10000
dirigeant . rémunération . net . après impôt: 10000
dirigeant . rémunération . totale: 14192
entreprise . charges: 1000
entreprise . chiffre d'affaires: 15192
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 5`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 42343
dirigeant . rémunération . cotisations: 19199
dirigeant . rémunération . net: 40719
dirigeant . rémunération . net . après impôt: 3000
dirigeant . rémunération . totale: 59918
entreprise . charges: 0
entreprise . chiffre d'affaires: 59916
impôt . montant: 4719

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 6`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 13954
dirigeant . rémunération . cotisations: 5559
dirigeant . rémunération . net: 13441
dirigeant . rémunération . net . après impôt: 13441
dirigeant . rémunération . totale: 19000
entreprise . charges: 1000
entreprise . chiffre d'affaires: 20000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > inversions 7`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 13210
dirigeant . rémunération . cotisations: 5276
dirigeant . rémunération . net: 12724
dirigeant . rémunération . net . après impôt: 12724
dirigeant . rémunération . totale: 18000
entreprise . charges: 2000
entreprise . chiffre d'affaires: 20000
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 1`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 549
dirigeant . rémunération . cotisations: 1409
dirigeant . rémunération . net: 500
dirigeant . rémunération . net . après impôt: 500
dirigeant . rémunération . totale: 1909
entreprise . charges: 0
entreprise . chiffre d'affaires: 1909
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 2`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 1065
dirigeant . rémunération . cotisations: 1500
dirigeant . rémunération . net: 1000
dirigeant . rémunération . net . après impôt: 1000
dirigeant . rémunération . totale: 2500
entreprise . charges: 0
entreprise . chiffre d'affaires: 2500
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 3`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 1581
dirigeant . rémunération . cotisations: 1589
dirigeant . rémunération . net: 1500
dirigeant . rémunération . net . après impôt: 1500
dirigeant . rémunération . totale: 3089
entreprise . charges: 0
entreprise . chiffre d'affaires: 3089
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 4`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 2097
dirigeant . rémunération . cotisations: 1679
dirigeant . rémunération . net: 2000
dirigeant . rémunération . net . après impôt: 2000
dirigeant . rémunération . totale: 3679
entreprise . charges: 0
entreprise . chiffre d'affaires: 3679
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 5`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 5193
dirigeant . rémunération . cotisations: 2217
dirigeant . rémunération . net: 5000
dirigeant . rémunération . net . après impôt: 5000
dirigeant . rémunération . totale: 7217
entreprise . charges: 0
entreprise . chiffre d'affaires: 7217
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 6`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 10382
dirigeant . rémunération . cotisations: 4192
dirigeant . rémunération . net: 10000
dirigeant . rémunération . net . après impôt: 10000
dirigeant . rémunération . totale: 14192
entreprise . charges: 0
entreprise . chiffre d'affaires: 14192
impôt . montant: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate simulations-indépendant > échelle de revenus 7`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 103815
dirigeant . rémunération . cotisations: 40624
dirigeant . rémunération . net: 100000
dirigeant . rémunération . net . après impôt: 73207
dirigeant . rémunération . totale: 140624
entreprise . charges: 0
entreprise . chiffre d'affaires: 140624
impôt . montant: 26793"
`;

exports[`calculate simulations-indépendant > échelle de revenus 8`] = `
"dirigeant . indépendant . cotisations et contributions . début activité: null
dirigeant . indépendant . revenu professionnel: 1033750
dirigeant . rémunération . cotisations: 243032
dirigeant . rémunération . net: 1000000
dirigeant . rémunération . net . après impôt: 528818
dirigeant . rémunération . totale: 1243032
entreprise . charges: 0
entreprise . chiffre d'affaires: 1243032
impôt . montant: 471182"
`;
